### DISCUSSION TITLE
Is it possible to change the output of the help pytest (--help)?

### DISCUSSION THREAD

**Discussion by GalacticBard08** [pytest-dev/pytest#10058] [Q&A]
There is a ready-made program that works with pytest, an ordinary user will work with it and for him and his colleagues it is necessary to create a help that would be called using --help, but this argument calls the pytest help and the necessary information about the added arguments used is at the very beginning of the end, which is not very convenient.

Is it possible to change the received help using the --help argument (you need to describe your functions and how to work with them)?

**Comment by The-Compiler** [comment]
Do you have a more concrete example? Why can't your tool just handle `--help` by itself instead of passing it to pytest?
